src/constants.ts

```diff
@@ -6,194 +6,89 @@ export const BSON_INT64_MAX = Math.pow(2, 63) - 1;
- **/
+/** Binary BSON Type */
 export const BSON_DATA_UNDEFINED = 6;
 
-/**
- * ObjectId BSON Type
- *
- * @classconstant BSON_DATA_OID
- **/
+/** ObjectId BSON Type */
 export const BSON_DATA_OID = 7;
 
-/**
- * Boolean BSON Type
- *
- * @classconstant BSON_DATA_BOOLEAN
- **/
+/** Boolean BSON Type */
 export const BSON_DATA_BOOLEAN = 8;
 
-/**
- * Date BSON Type
- *
- * @classconstant BSON_DATA_DATE
- **/
+/** Date BSON Type */
 export const BSON_DATA_DATE = 9;
 
-/**
- * null BSON Type
- *
- * @classconstant BSON_DATA_NULL
- **/
+/** null BSON Type */
 export const BSON_DATA_NULL = 10;
 
-/**
- * RegExp BSON Type
- *
- * @classconstant BSON_DATA_REGEXP
- **/
+/** RegExp BSON Type */
 export const BSON_DATA_REGEXP = 11;
 
-/**
- * Code BSON Type
- *
- * @classconstant BSON_DATA_DBPOINTER
- **/
+/** Code BSON Type */
 export const BSON_DATA_DBPOINTER = 12;
 
-/**
- * Code BSON Type
- *
- * @classconstant BSON_DATA_CODE
- **/
+/** Code BSON Type */
 export const BSON_DATA_CODE = 13;
 
-/**
- * Symbol BSON Type
- *
- * @classconstant BSON_DATA_SYMBOL
- **/
+/** Symbol BSON Type */
 export const BSON_DATA_SYMBOL = 14;
 
-/**
- * Code with Scope BSON Type
- *
- * @classconstant BSON_DATA_CODE_W_SCOPE
- **/
+/** Code with Scope BSON Type */
 export const BSON_DATA_CODE_W_SCOPE = 15;
 
-/**
- * 32 bit Integer BSON Type
- *
- * @classconstant BSON_DATA_INT
- **/
+/** 32 bit Integer BSON Type */
 export const BSON_DATA_INT = 16;
 
-/**
- * Timestamp BSON Type
- *
- * @classconstant BSON_DATA_TIMESTAMP
- **/
+/** Timestamp BSON Type */
 export const BSON_DATA_TIMESTAMP = 17;
 
-/**
- * Long BSON Type
- *
- * @classconstant BSON_DATA_LONG
- **/
+/** Long BSON Type */
 export const BSON_DATA_LONG = 18;
 
-/**
- * Long BSON Type
- *
- * @classconstant BSON_DATA_DECIMAL128
- **/
+/** Decimal128 BSON Type */
 export const BSON_DATA_DECIMAL128 = 19;
 
-/**
- * MinKey BSON Type
- *
- * @classconstant BSON_DATA_MIN_KEY
- **/
+/** MinKey BSON Type */
 export const BSON_DATA_MIN_KEY = 0xff;
 
-/**
- * MaxKey BSON Type
- *
- * @classconstant BSON_DATA_MAX_KEY
- **/
+/** MaxKey BSON Type */
 export const BSON_DATA_MAX_KEY = 0x7f;
 
-/**
- * Binary Default Type
- *
- * @classconstant BSON_BINARY_SUBTYPE_DEFAULT
- **/
+/** Binary Default Type */
 export const BSON_BINARY_SUBTYPE_DEFAULT = 0;
 
-/**
- * Binary Function Type
- *
- * @classconstant BSON_BINARY_SUBTYPE_FUNCTION
- **/
+/** Binary Function Type */
 export const BSON_BINARY_SUBTYPE_FUNCTION = 1;
 
-/**
- * Binary Byte Array Type
- *
- * @classconstant BSON_BINARY_SUBTYPE_BYTE_ARRAY
- **/
+/** Binary Byte Array Type */
 export const BSON_BINARY_SUBTYPE_BYTE_ARRAY = 2;
 
-/**
- * Binary UUID Type
- *
- * @classconstant BSON_BINARY_SUBTYPE_UUID
- **/
+/** Binary Deprecated UUID Type @deprecated Please use BSON_BINARY_SUBTYPE_UUID_NEW */
 export const BSON_BINARY_SUBTYPE_UUID = 3;
 
-/**
- * Binary MD5 Type
- *
- * @classconstant BSON_BINARY_SUBTYPE_MD5
- **/
-export const BSON_BINARY_SUBTYPE_MD5 = 4;
```

Comment:
We don't use this anywhere internally, but we do export it.
I added tests to make sure we have the correct values, who knows typos happen best to guard against that.